plugins/ltac/g_obligations.ml4

```diff
@@ -6,11 +6,11 @@
 (*         *       GNU Lesser General Public License Version 2.1        *)
 (************************************************************************)
 
-(*i camlp4deps: "grammar/grammar.cma" i*)
+(*i camlp5deps: "grammar/grammar.cma" i*)
 
 (*
   Syntax for the subtac terms and types.
-  Elaborated from correctness/psyntax.ml4 by Jean-Christophe Filli�tre *)
+  Elaborated from correctness/psyntax.ml4 by Jean-Christophe Filli�tre *)
```

Comment:
You seem to have an encoding issue. Probably due to Windows UTF-16. Keep to UTF-8 please.

Reply:
I'm not sure what is happening.
- My local git shows no change to this line
- https://github.com/jfehrle/coq/blob/master/plugins/ltac/g_obligations.ml4 shows the â rather than the �.
- When I examine the file with od, it contains the 3 byte sequence for �.  My editor shows the �, which I can replace with â, which then appears as a changed line in git.  That should fix it.

� is encoded as ef bf bd while â is c3 a2.  The former is actually in the file on my system:

```
0001160   s   t   o   p   h   e       F   i   l   l   i 357 277 275   t

0001160 73 74 6f 70 68 65 20 46 69 6c 6c 69 ef bf bd 74
```

FWIW: UTF-16 is mostly used in Asian countries with languages that have a very large character set such as Japanese and Chinese.  I've never encountered it.  Windows programs frequently use UTF-8 or one of the 8-bit ISO 8859 encodings.  This is clearly UTF-8, just this character is getting substituted somehow.

Reply:
Whoops, if I look in my branch cleanup_coq at https://github.com/jfehrle/coq/blob/master/plugins/ltac/g_obligations.ml4 it shows the �.

Reply:
emacs opens that file in iso-latin1 on my computer for master.